pipeline-model-definition/src/main/resources/org/jenkinsci/plugins/pipeline/modeldefinition/ClosureModelTranslator.groovy

```diff
@@ -46,6 +46,8 @@ import static org.jenkinsci.plugins.pipeline.modeldefinition.Utils.createStepsBl
  * @author Andrew Bayer
  */
 public class ClosureModelTranslator implements MethodMissingWrapper, Serializable {
+    private static final List<Class> UNTRANSLATED_CLASSES = [StageConditionals.class]
```

Comment:
Making this a `List` to check against since we're doing this again for `Environment` over in #110, and I imagine there'll be more in the future as I gradually kill off the translators.